crm_rma_prodlot_supplier/models/stock_production_lot.py

```diff
@@ -38,15 +38,18 @@ class StockProductionLot(models.Model):
 
     @api.model
     def default_get(self, fields_list):
-        """
-        Set partner when product lot is created
+        """ Set partner when product lot is created
         @param fields_list: record values
         @return: return record
         """
         res = super(StockProductionLot, self).default_get(fields_list)
 
         prodlot_obj = self.env['stock.production.lot']
-        transfer_item_id = self._context.get('active_id', False)
+
+        transfer_item_id = False
+        if self._context.get('active_model', '') ==\
```

Comment:
In #90 this change is also done, in another way, and might give a merge request now..